rosdep/base.yaml

```diff
@@ -8966,4 +8966,4 @@ zziplib:
   nixos: [zziplib]
   opensuse: [zziplib-devel]
   rhel: [zziplib-devel]
-  ubuntu: [libzzip-0-13, libzzip-dev]
+  ubuntu: [libzzip-dev]
```

Comment:
While we are here, we should do the same for the `debian` key above.

Reply:
Done https://github.com/ros/rosdistro/pull/40304/commits/56f87f672afa34f140eb7e6401e25d98582f068c